drivers/dummy/rasterizer_dummy.h

```diff
@@ -182,7 +182,7 @@ class RasterizerStorageDummy : public RasterizerStorage {
 
 		ERR_FAIL_COND(!t);
 		ERR_FAIL_COND(t->format != p_image->get_format());
-		ERR_FAIL_COND(p_image.is_null());
+		ERR_FAIL_COND_MSG(p_image.is_null(), "Image cannot be empty.");
```

Comment:
See https://github.com/godotengine/godot/pull/32051/files#r326995001

This check might be a bit bogus though, it should likely happen before line 184.